src/app/shared/modal/share-view/body/share-view-dialog-body.component.scss

```diff
@@ -1,3 +1,7 @@
 table {
   table-layout: fixed;
 }
+
+.tools {
+  width: 8.75rem;
```

Comment:
Are you sure about that? Dialog can have various width, and especially on mobile devices this value may be very big.